### ISSUE TITLE
Can the imagenet224_conv_16 model train directly, without finetune?

### ISSUE THREAD

**Issue by terrychenism** [abangdd/nobodyDL#1]
We know if we need train vgg16, we need train vgg11 first,  finetune to vgg13, then finetune to VGG16.  I check your model, which is very similar to vgg16. Can this model train directly without finetune from small network? Could you provide some training parameters, like lr, stepsize, weight decay for me?

**Comment by abangdd** [comment]
yes, you can adpot the method in the paper "Delving Deep into Rectifiers", but it will be a little bit harder than my model or msra model, maybe because vgg model has one more max pooling layers, so you will need to try several times and see if it works
I use gradient exploding or vanishing check in my code, but sometimes the 5 max pooling layer model does not converge, I see the softmax output does not change, so luck is needed

**Comment by abangdd** [comment]
I always train model directly and always use the largest lr in the first epoch unless the log says gradient exploding
lr must be slowly lowered every epoch, like this code

{ const float a = max_round;
  const float b = now_round;
  const float r = lr_last / lr_base;
  const float x = log (r) / log (1.f/a);
  lrate = lr_base * pow (1.f - b/a, x)

I adopt this lr policy in Caffe, perhaps namely "poly"

weight decay needs to be tried for every dataset, but for the same dataset+different model, there is no need to try again and again
